site/content/docs/02-template-syntax.md

```diff
@@ -141,6 +141,13 @@ Text can also contain JavaScript expressions:
 <p>{a} + {b} = {a + b}.</p>
 ```
 
+---
+
+However, if you're using a regular expression (`RegExp`) [literal notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#literal_notation_and_constructor), you'll need to wrap it in parentheses.
```

Comment:
```suggestion
> If you're using a regular expression (`RegExp`) [literal notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#literal_notation_and_constructor), you'll need to wrap it in parentheses.
```

Reply:
Placing it in an aside box seems nice. I haven't seen what it'll look like on the page if we do that, but I see in the current docs that blockquotes are always placed after a main text first. Should we also remove the separator above? What about the example code after it?

Reply:
This blue box is an example: https://svelte.dev/docs#get

Yes, I think we could remove the separator in that case. Can we put the code example in the box? (maybe if you prefix each line with `>` it will do that. you can test by running the site locally, which I think is `cd site; npm install; npm run update; npm run dev`)

Reply:
It looks a bit weird

![image](https://user-images.githubusercontent.com/8156777/125020812-b96ab000-e0a3-11eb-8568-9c42139dcbd3.png)